web/js/funcbtn.js

```diff
@@ -0,0 +1,28 @@
+// Get the modal
```

Comment:
Very strange name for file. It is impossible to understand what is in this file by its name. Name it something like "dialogHandler.js"